app/services/moderator/sink_user.rb

```diff
@@ -0,0 +1,35 @@
+module Moderator
+  class SinkUser
+    def self.call(reaction:)
+      new(reaction: reaction).vomit_all_articles
+    end
+
+    def self.confirm(reaction:)
+      new(reaction: reaction).confirm_vomits
+    end
+
+    def initialize(reaction:)
+      @reaction = reaction
+      @user = reaction.user
+      @articles = reaction.reactable.articles
+    end
+
+    attr_reader :user, :reaction, :articles
+
+    def vomit_all_articles
+      reaction_objects = articles.map { |article| Reaction.new(reactable: article, user: user, category: "vomit", status: "bulk_submitted") }
+      Reaction.import reaction_objects
+    end
+
+    def confirm_vomits
+      vomit_reactions = Reaction.where(reactable: articles, status: "bulk_submitted", user: user, category: "vomit")
+      vomit_reactions.update_all(status: "confirmed")
```

Comment:
Okay, so I think `.update_all` here is the most efficient solution for sure. However we use a lot of callbacks in the Reaction model, for example:
```ruby
  after_save :index_to_algolia
  after_save :update_reactable, :bust_reactable_cache, :touch_user
```

Fairly certain these are pretty important callbacks. I was thinking I could simply run the callbacks after (with the exception of `touch_user` which will only need to be run once at the very end). Not sure if that's the best/most efficient way to go about this though?